.github/workflows/loki.yaml

```diff
@@ -14,26 +14,31 @@ on:
       - Cilium L4LB XDP (ci-l4lb-1.11)
       - Cilium L4LB XDP (ci-l4lb-1.12)
       - Cilium L4LB XDP (ci-l4lb-1.13)
+      - Cilium L4LB XDP (ci-l4lb-1.14)
```

Comment:
[Useless nit] The `ci-xxx-1.xx` workflows do not exist anymore, now it's only the regular non `-1.xx` versions for all branches. Not that matters considering Loki is disabled anyway...

Reply:
Yeah I was debating if we should just be deleting the loki workflow instead.

Reply:
https://github.com/cilium/cilium/pull/26676